test/controllers/products_controller_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
+
```

Comment:
@roseaboveit Thank you for the note about if/else path testing. We didn't have time to cover controller testing in class, and our instructor told us to not worry about controller testing due to that. :)